cmake/Modules_CUDA_fix/upstream/FindCUDA/select_compute_arch.cmake

```diff
@@ -192,14 +192,10 @@ function(CUDA_SELECT_NVCC_ARCH_FLAGS out_variable)
       set(arch_ptx ${arch_bin})
     else()
       # Look for it in our list of known architectures
-      if(${arch_name} STREQUAL "Fermi")
-        set(arch_bin 2.0 "2.1(2.0)")
-      elseif(${arch_name} STREQUAL "Kepler+Tegra")
-        set(arch_bin 3.2)
-      elseif(${arch_name} STREQUAL "Kepler+Tesla")
```

Comment:
could be changed to a check that gives an error later. since `arch_bin` can have one or multiple values, not sure that's more robust. could also be left alone - it works today, will just error somewhere down the line if anyone tries to use it